app/models/journeyDomain/routeDetails/locationOfGoods/LocationOfGoodsDomain.scala

```diff
@@ -31,6 +31,8 @@ sealed trait LocationOfGoodsDomain extends JourneyDomainModel {
 
   val qualifierOfIdentification: LocationOfGoodsIdentification
 
+  val contactPerson: Option[AdditionalContactDomain]
```

Comment:
Minor suggestion but could put `val additionalContact: Option[AdditionalContactDomain] = None` here and then `override val additionalContact: Option[AdditionalContactDomain]` in the case classes that need it